kroxylicious-filters/kroxylicious-authorization/src/main/java/io/kroxylicious/filter/authorization/DeleteTopicsEnforcement.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.filter.authorization;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import org.apache.kafka.common.Uuid;
+import org.apache.kafka.common.message.DeleteTopicsRequestData;
+import org.apache.kafka.common.message.DeleteTopicsResponseData;
+import org.apache.kafka.common.message.RequestHeaderData;
+import org.apache.kafka.common.protocol.Errors;
+
+import io.kroxylicious.authorizer.service.Action;
+import io.kroxylicious.authorizer.service.AuthorizeResult;
+import io.kroxylicious.authorizer.service.Decision;
+import io.kroxylicious.proxy.filter.FilterContext;
+import io.kroxylicious.proxy.filter.RequestFilterResult;
+import io.kroxylicious.proxy.filter.metadata.TopicNameMapping;
+import io.kroxylicious.proxy.filter.metadata.TopicNameMappingException;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+class DeleteTopicsEnforcement extends ApiEnforcement<DeleteTopicsRequestData, DeleteTopicsResponseData> {
+
+    @Override
+    short minSupportedVersion() {
+        return 1;
+    }
+
+    @Override
+    short maxSupportedVersion() {
+        // DELETE_TOPICS: v6 allows topic ids: It's better to force the client to use v5 than fail a v6 later on
```

Comment:
I don't understand the comment.  You say use v5, then set v6.  Out of date comment?

Reply:
From conversation outside of GitHub the comment is stale. There is support here and only here for topicIds.

Reply:
Yes.